charts/victoria-metrics-agent/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## Next release
 
+- Use only podLabels as extra labels for pods
+- Renamed `statefulset` to `statefulSet` for consistency
```

Comment:
```suggestion
- Renamed `.Values.statefulset` to `.Values.statefulSet` for consistency
```
I'd suggest adding `.Values` prefix to make it more explicit that the change requires values file update.